components/NavBar/index.js

```diff
@@ -40,12 +41,22 @@ const NavBar = ({ personData: { photo } }) => {
         </div>
       </li>
 
-      <div
-        className={styles.profilePic}
-        ref={navbarRef}
-        onClick={() => setToggle(!toggle)}
-      >
-        <img src={photo} alt="Profile Image" height="70" width="70" />
+      {/* <li>
+        <Link href="#">
+          <a className={styles.loginBtn}>Log in</a>
+        </Link>
+      </li> */}
```

Comment:
Are we keeping commented code?

Reply:
I'm not sure I don't remember Commenting that code out. Probably not.

Reply:
Let's get rid of commented code please

Reply:
Still working on this one?